mce-label-injector/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Build the manager binary
+FROM golang:1.23 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY webhook/ webhook/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o mce-label-injector main.go
+
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static:nonroot
+WORKDIR /
+COPY --from=builder /workspace/mce-label-injector /usr/local/bin
+USER nonroot:nonroot
+
+ENTRYPOINT ["/usr/local/bin/mce-label-injector"]
```

Comment:
better naming instead of **mce-lable-injector** use **mce-capi-mutating-webhook-config** OR **mce-capi-webhook-config** . For directory name, service name, image name, ...etc